defaults/config.js

```diff
@@ -51,9 +51,18 @@ module.exports = {
 	// Set the default theme.
 	//
 	// @type     string
-	// @default  "themes/example.css"
+	// @default  "example"
 	//
-	theme: "themes/example.css",
+	theme: "example",
+
+	//
+	// A list of plugins installed as global npm modules
+	//
+	// Find available themes at https://www.npmjs.com/search?q=keywords%3Alounge-theme
```

Comment:
Stupid question but since the npm package is called `thelounge`, do we want to refer to `thelounge-theme` to match the package name, or `lounge-theme` to match the command name?

Reply:
Yeah, can do.

Reply:
This inconsistency is quite annoying. Repo is called `lounge`, that search link is looking for a keyword, not package name.

Reply:
Yeah...I agree. We do have issues in having the 2 names.